dotnet/simple-app/Program.cs

```diff
@@ -0,0 +1,194 @@
+*/
+
+using System.Text;
+using ImmuDB;
+using ImmuDB.Exceptions;
+
+namespace simple_app;
+
+class Program
+{
+    public static async Task Main(string[] args)
+    {
+        await InitializeExample();
+        await AnotherInitializeExample();
+        await YetAnotherInitializeExample();
+        await MiscFunctionsUsageExample();
+        await ImmuClient.ReleaseSdkResources();
+    }
+
+    private static async Task InitializeExample()
+    {
+        var client = ImmuClient.NewBuilder().Build();
+        await client.Open("immudb", "immudb", "defaultdb");
+
+        string key = "hello";
+
+        try
+        {
+            // Setting (adding) a key-value.
+            await client.Set(key, "test");
+
+            // Getting it back, by key (in a verified way that reports any tampering if it happened).
+            Entry entry = await client.VerifiedGet(key);
+            Console.WriteLine($"{key}, {entry.Value.ToString()}");
+        }
+        catch (VerificationException e)
+        {
+            // VerificationException means Data Tampering detected!
+            // This means the history of changes has been tampered.
+            Console.WriteLine(e.ToString());
+        }
+
+        await client.Close();
+    }
+    
+    private static async Task AnotherInitializeExample()
+    {
+        var client = await ImmuClient.NewBuilder().Open();
+        string key = "hello";
+
+        try
+        {
+            // Setting (adding) a key-value.
+            await client.Set(key, "immutable world!");
+
+            // Getting it back, by key (in a verified way that reports any tampering if it happened).
+            Entry entry = await client.VerifiedGet(key);
+            Console.WriteLine($"{key}, {entry.ToString()}");
+        }
+        catch (VerificationException e)
+        {
+            // VerificationException means Data Tampering detected!
+            // This means the history of changes has been tampered.
+            Console.WriteLine(e.ToString());
+        }
+
+        await client.Close();
+    }
+
+    private static async Task YetAnotherInitializeExample()
+    {
+        var client = new ImmuClient();
+        await client.Open("immudb", "immudb", "defaultdb");
+        string key = "hello";
+
+        try
+        {
+            // Setting (adding) a key-value.
+            await client.Set(key, "immutable world!");
+
+            // Getting it back, by key (in a verified way that reports any tampering if it happened).
+            Entry entry = await client.VerifiedGet(key);
+            Console.WriteLine($"{key}, {entry.ToString()}");
+        }
+        catch (VerificationException e)
+        {
+            // VerificationException means Data Tampering detected!
+            // This means the history of changes has been tampered.
+            Console.WriteLine(e.ToString());
+        }
+
+        await client.Close();
+    }
+
+    private static async Task MiscFunctionsUsageExample()

```

Comment:
What does misc mean ? This one example covers the above examples, I would leave only it then ?

Reply:
Misc = miscellaneous. That is not true. The way client is constructed differs.

Reply:
Removed the confusing name "misc".